apps/meteor/server/methods/registerUser.js

```diff
@@ -94,7 +95,7 @@ Meteor.methods({
 			throw new Meteor.Error(e.message);
 		}
 
-		Users.setName(userId, trim(formData.name));
+		Users.setName(userId, escapeHtml(trim(formData.name)));
```

Comment:
so the idea here is saving a different name that I actually set? save an original name with a escaped value?
it doesn’t look good to me, the places where we render the name should handle that, its not the easiest way but the right one to do

maybe if its the case we should not allow such characters 

Anyway you are only handling the user registration but not later if the user changes the name using the api